modules/nomad-vault-integration/main.tf

```diff
@@ -1,20 +1,16 @@
-#################################################
-# AWS Authentication
-#################################################
-resource "vault_auth_backend" "aws" {
-  type = "aws"
-  path = "${var.aws_auth_path}"
-}
+# Basaed on https://www.nomadproject.io/docs/vault-integration/index.html
 
 ################################################
 # Nomad Server Policy
-# See https://www.nomadproject.io/docs/vault-integration/index.html
+# This policy allows creation of a periodic token to pass to Nomad servers using the
+# `nomad_server_role`.
+# This policy has to be attached to the token that your instance has
```

Comment:
Weird phrasing, maybe something like "attached to the token that was obtained/generated by your instance"?

Reply:
Fixed.